tests/testApproximate.py

```diff
@@ -53,7 +53,8 @@ def makeRamp(self, binsize=1):
         rampCoeffs = (1000, 1, 1)
         for i in range(ramp.getHeight()):
             for j in range(ramp.getWidth()):
-                ramp.set(j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
+                ramp.set(
+                    j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
```

Comment:
IMO, "could" and "should" are very different things here.  Formatting new code with autopep8 is fine; formatting old code that may have been hand-formatted to make an important piece of logic readable is not okay, and we should neither assume it will happen nor encourage it.